roaring/btree.go

```diff
@@ -0,0 +1,960 @@
+// This file is a modified redistribution of b (https://github.com/cznic/b),
```

Comment:
We should come up with some way of monitoring the original btree repo for any bugfixes or changes which might be relevant to us. I'm now "watching" the repo on github, so I'll get emails whenever there are changes. This shouldn't be too burdensome as it's a small repo that doesn't appear to be undergoing active dev (just maintenance).